client/state/action-types.js

```diff
@@ -6,6 +6,8 @@
  */
 
 export const FAIL_PUBLICIZE_CONNECTIONS_REQUEST = 'FAIL_PUBLICIZE_CONNECTIONS_REQUEST';
+export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
+export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
```

Comment:
How about using `REQUEST` and `REPLY` instead of `FETCH` and `RECEIVE`? Since not all API calls are necessarily 'fetching' data - for example [triggering an asynchronous task](https://github.com/Automattic/wp-calypso/pull/1126/files#diff-c53625f942da257ecda4f9f5bea55ebdR17). `FETCH_START_EXPORT` and `RECEIVE_START_EXPORT` don't really make sense, but `REQUEST_START_EXPORT` and `REPLY_START_EXPORT` describe the action somewhat more clearly.